Fedora-35/Dockerfile

```diff
@@ -56,15 +56,36 @@ RUN npm install -g npm markdownlint-cli cspell
 # This image is indented for jobs that run tests (and possibly also build)
 # firmware images. It is based on the build image and adds Qemu for the
 # architectures under test.
+
+#Building qemu from source:
 FROM build AS test
+ENV QEMU_URL="https://download.qemu.org/qemu-7.1.0.tar.xz"
 RUN microdnf \
       --assumeyes \
       --nodocs \
       --setopt=install_weak_deps=0 \
       install \
-        qemu-system-aarch64-core \
-        qemu-system-arm-core \
-        qemu-system-x86-core
+        bzip2 \
+        findutils \
+        git \
+        glib2-devel \
+        libfdt-devel \
+        ninja-build \
+        pixman-devel \
+        python3 \
+        tar \
+        zlib-devel
+RUN curl -o qemu.tar.xz "${QEMU_URL}"
```

Comment:
I wonder if the download + build + delete should all be one RUN but with && statements to avoid having intermediate steps increase the total image size.